plugins/catalog-backend/src/schema/openapi.yaml

```diff
@@ -850,6 +850,10 @@ paths:
                   type: array
                   items:
                     type: string
+                filter:
+                  type: array
+                  items:
+                    type: string
```

Comment:
Hm. This format was made the way it was because of limited expressivity of HTTP query parameters. Does it really make much sense to serialize the `EntityFilterQuery` like this when it's going to be carried in a JSON body anyway? It looks like a smell kinda.

The reason I'm asking is, is it time to actually bite the bullet and choose a more purpose built format for filters when used in JSON bodies? Like for example the actual receiving end's `EntityFilter` with its `anyOf` etc.

If not, I'm almost tempted to just put this as a query parameter even on this POST (instead of the body) as a first step, so we don't have to invent anything anywhere for now, and then we can move forward with more purpose built things at a later time - including using that format in a POST form of `queryEntities` as well. Then that's a separate effort to be considered in isolation, possibly even accompanied with a bep for "the next generation query format for the catalog".